starter-code/styles/base.css

```diff
@@ -1,4 +1,7 @@
 /* TODO: Add a rule for images, so they never exceed their container's width. */
+img {
+  min-width: 100%;
```

Comment:
So with this, you are setting the minimum with of an img to be 100%, but you are also giving it a possibility to get bigger, which is not what we want. You dont want to target the min but the ... I'll let you guys figure that out and resubmit for the point